### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sidsharma27**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,430 @@
...
+    if (controlButtons.isHidden) {
+        controlButtons.isHidden = false;
+        perform(#selector(hideControlButtons), with:nil, afterDelay:3)
+    }
+}
+```
 
-The MIT License (MIT).
+## Resources
+If you have any questions, please feel free to reach out via [e-mail](mailto:sid.sharma@agora.io) or [Twitter](https://twitter.com/sidsharma_27).
```
Comment (guohai) [inline README.md]: Can not forward to personal email or twitter

File: README.md [hunk 2]
```diff
@@ -1,47 +1,430 @@
...
-Agora Video SDK supports iOS / Android / Windows / macOS etc. You can find demos of these platform here:
+### Build the Sample Application 
 
-- [Agora-Android-Tutorial-1to1](https://github.com/AgoraIO/Agora-Android-Tutorial-1to1)
-- [Agora-Windows-Tutorial-1to1](https://github.com/AgoraIO/Agora-Windows-Tutorial-1to1)
-- [Agora-macOS-Tutorial-Swift-1to1](https://github.com/AgoraIO/Agora-macOS-Tutorial-Swift-1to1)
+1. Clone this repository.
+2. Open the project file **Agora iOS Tutorial.xcodeproj** in Xcode.
+3. In the Project Navigator, navigate to **Agora iOS Tutorial** > **Agora iOS Tutorial**.
+4. Open **VideoCallViewController.swift**.
```
Comment (PRWrl) [inline README.md]: the appId is in AppID.swift

File: README.md [hunk 3]
```diff
@@ -1,47 +1,430 @@
...
+
+## Develop the Application 
+
+This workflow was used to develop the 1-to-1 sample application:
+
+1. [Set up the project](#set-up-the-project)
+2. [Configure Resources](#configure-resources)
+3. [Manage Communication Features](#manage-communication-features)
+
+For details about the APIs used to develop this sample, see the [Agora.io Documentation](https://docs.agora.io/en/2.2).
```
Comment (PRWrl) [inline README.md]: maybe not specific to 2.2 ?

File: README.md [hunk 4]
```diff
@@ -1,47 +1,430 @@
...
+|`switchCameraButton`|A picture of a camera and rotational arrows to switch between the two cameras.|
+|`switchCameraButtonSelected`|A highlighted picture of a camera and rotational arrows to indicate that the rear camera is in use.|
+|`videoMuteButton`|A picture of a camera to mute video.|
+|`videoMuteButtonSelected`|A picture of a camera highlighted to indicate that video is muted.|
+|`videoMuteButtonIndicator`|A picture of a camera crossed out to indicate the camera is off.|
+
+
+
+### Design the User Interface
+The sample was developed using the [model-view-controller](https://en.wikipedia.org/wiki/Model–view–controller) pattern. The *Main.storyboard* file defines the user interface and makes use of two view controllers: *VideoCallViewController.swift* and *SetChannelViewController.swift*.
```
Comment (PRWrl) [inline README.md]: this tutorial doesn't have SetChannelViewController.swift

**Comment by sidsharma27** [comment]
@PRWrl plz check now

File: README.md [hunk 5]
```diff
@@ -1,47 +1,391 @@
...
+pod init
+```
+
+Modify the new pod file to include the `AgoraRtcEngine_iOS` pod:
+
+```bash
+target 'AgoraVideoQuickstart' do
+  use_frameworks!
+  pod 'AgoraRtcEngine_iOS'
+end
```
Comment (PRWrl) [inline README.md]: This project doesn't use pod, have to copy frameworks manually

File: README.md [hunk 6]
```diff
@@ -1,47 +1,391 @@
...
+}
+```
+
+### Configure Video Mode
+The next step is to enable video mode, configure the video encoding profile, and specify if the width and height can change when switching from portrait to landscape:
+
+``` swift
+func setupVideo() {
+    agoraKit.enableVideo()  // Enables video mode.
+    agoraKit.setVideoProfile(._VideoProfile_360P, swapWidthAndHeight: false) // Default video profile is 360P
```
Comment (PRWrl) [inline README.md]: After v2.1.0, this should be 
agoraKit.setVideoProfile(.portrait360P, swapWidthAndHeight: false)

or use new API sine v2.3.0
`
agoraKit.setVideoEncoderConfiguration(AgoraVideoEncoderConfiguration(size: AgoraVideoDimension640x360,
                                                                     frameRate: .fps15,
                                                                     bitrate: AgoraVideoBitrateStandard,
                                                                     orientationMode: .adaptative))
`

File: README.md [hunk 7]
```diff
@@ -1,47 +1,391 @@
...
+
+### Set up Local Video
+The logic for the local video feed is contained within a helper method called `setupLocalVideo()`, which is invoked by `viewDidLoad()`:
+
+``` swift
+func setupLocalVideo() {
+    let videoCanvas = AgoraRtcVideoCanvas()
+    videoCanvas.uid = 0
+    videoCanvas.view = localVideo
+    videoCanvas.renderMode = .render_Fit
```
Comment (PRWrl) [inline README.md]: videoCanvas.renderMode = .fit

File: README.md [hunk 8]
```diff
@@ -1,47 +1,391 @@
...
+    setupVideo();
+    setupLocalVideo();   
+}
 ```
-let AppID: String = "Your App ID"
+
+`setupLocalVideo()` creates an `AgoraRtcVideoCanvas` object for the video stream and initializes the following properties:
+* **uid**: A value of 0 allows Agora to chose a random ID for the stream feed.
+* **view**: Set to the `localVideo` view from the storyboard.
+* **renderMode**: Set to `render_Fit` to ensure the video is resized proportionally to fit the display window.
```
Comment (PRWrl) [inline README.md]: Set to `fit`

File: README.md [hunk 9]
```diff
@@ -1,47 +1,397 @@
...
+
+
+### Design the User Interface
+The sample was developed using the [model-view-controller](https://en.wikipedia.org/wiki/Model–view–controller) pattern. The *Main.storyboard* file defines the user interface and makes use of one view controller: *VideoCallViewController.swift*.
+
+#### Prepare the Video Call View Controller
+*VideoCallViewController.swift* defines a view that handles a video call. Note that its file *VideoCallViewController.swift* was renamed from the default file name *ViewController.swift* to reflect the purpose of the view.
+
+These are the main aspects of the Video Call View Controller on the storyboard:
+![Storyboard_Setup2.png](/StoryboardSetup.png)
```
Comment (PRWrl) [inline README.md]: img missing

File: README.md [hunk 10]
```diff
@@ -1,47 +1,397 @@
...
+
+Install the pod file:
+
+```bash
+pod install
+```
+
+### Set Permissions
+In the `Info.plist` file, set the privacy settings for the camera and microphone to **Video Chat** to allow the app to access them:
+![Info_Plist.png](/Info_Plist.png)
```
Comment (PRWrl) [inline README.md]: img missing

**Comment by sidsharma27** [comment]
Added images

**Pull request merged**